gems/user_agent_parser/GHSA-pcqq-5962-hvcw.yml

```diff
@@ -0,0 +1,24 @@
+---
+gem: user_agent_parser
+ghsa: pcqq-5962-hvcw
+url: https://github.com/ua-parser/uap-ruby/security/advisories/GHSA-pcqq-5962-hvcw
```

Comment:
The GHSA is for `uap-ruby`, not `user_agent_parser`. If the GHSA/CVE was for a non-rubygem-package, we can safely ignore it.